examples/land-cover-fastai/deep_lulc_walkthrough.ipynb

```diff
@@ -75,7 +75,16 @@
    "cell_type": "code",
```

Comment:
Is this part still relevant?

![image](https://user-images.githubusercontent.com/37869250/88773498-7a896600-d182-11ea-992f-5c8e61c0d51a.png)

Reply:
yes, I think it is best to leave this in